src/lib/chat/index.ts

```diff
@@ -80,6 +80,8 @@ export interface IChatClient {
   generateSecureBackup: () => Promise<any>;
   saveSecureBackup: (MatrixKeyBackupInfo) => Promise<void>;
   restoreSecureBackup: (recoveryKey: string) => Promise<void>;
+  getRoomIdForAlias: (alias: string) => Promise<string | undefined>;
+  apiJoinRoom: (aliasOrId: string) => Promise<{ success: boolean; response: any; message: string }>;
```

Comment:
We really don't need to keep adding to this interface since we only have a single client now. Ok to leave in but for the future.

Reply:
got it.